libraries/chain/database/database.cpp

```diff
@@ -210,15 +221,15 @@ void database::reindex(const fc::path& data_dir,
 
         with_write_lock([&]() {
             auto itr = _block_log.read_block(0);
-            auto last_block_num = _block_log.head()->block_num();
-
             while (itr.first.block_num() != last_block_num)
             {
                 auto cur_block_num = itr.first.block_num();
-                if (cur_block_num % 100000 == 0)
-                    std::cerr << "   " << double(cur_block_num * 100) / last_block_num << "%   " << cur_block_num
-                              << " of " << last_block_num << "   (" << (get_free_memory() / (1024 * 1024))
-                              << "M free)\n";
+                if (cur_block_num % log_interval_sz == 0)
```

Comment:
Add ` || cur_block_num == last_block_num` condition here so we log info for the rest of blocks